sdk/python/Makefile.toml

```diff
@@ -1,17 +1,25 @@
 [tasks.maturin]
+dependencies = ["pyenv"]
 script = '''
 source .env/bin/activate
 maturin develop
 '''
 
 [tasks.ipython]
+dependencies = ["pyenv"]
 script = '''
 source .env/bin/activate
 ipython
 '''
 
 [tasks.python]
+dependencies = ["pyenv"]
 script = '''
 source .env/bin/activate
 python "${@}"
 '''
+
+[tasks.pyenv]
```

Comment:
what is this for?

Reply:
@YikSanChan to generate `.env` directory using `python -m venv .env` if not exists